spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMigrateTableProcedure.java

```diff
@@ -122,6 +122,27 @@ public void testMigrateWithDropBackup() throws IOException {
     Assert.assertFalse(spark.catalog().tableExists(tableName + "_BACKUP_"));
   }
 
+  @Test
+  public void testMigrateWithBackupTableName() throws IOException {
+    Assume.assumeTrue(catalogName.equals("spark_catalog"));
+    String location = temp.newFolder().toString();
+    sql(
+        "CREATE TABLE %s (id bigint NOT NULL, data string) USING parquet LOCATION '%s'",
+        tableName, location);
+    sql("INSERT INTO TABLE %s VALUES (1, 'a')", tableName);
+
+    String backupTableName = "backup_table";
+    Object result =
+        scalarSql(
+            "CALL %s.system.migrate(table => '%s', backup_table_name => '%s')",
+            catalogName, tableName, backupTableName);
+
+    Assert.assertEquals("Should have added one file", 1L, result);
+
+    String dbName = tableName.split("\\.")[0];
+    Assert.assertTrue(spark.catalog().tableExists(dbName + "." + backupTableName));
```

Comment:
Please use `AssertJ` instead. You can refer the contributing guide: https://iceberg.apache.org/contribute/#assertj

Reply:
Thanks for the advice. Replace the test part with it.